### PR TITLE
add init default SM CMK

### PR DESCRIPTION
**Pull request by Yidingluo1**
add default CMK when enroll user info

Signed-off-by: luoyiding <luo.yd@neusoft.com>

### CODE REVIEW

File: ehsm_kms_service/function.js [hunk 1]
```diff
@@ -264,8 +264,10 @@ const enroll_user_info = (action, DB, res, req) => {
   if (napi_res) {
     const { appid, apikey } = napi_res.result
     let cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
+    let sm_default_cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
```
Comment (yang8621) [inline ehsm_kms_service/function.js:267]: will the result be checked?

Comment (Yidingluo1) [reply ehsm_kms_service/function.js:267]: Yes, it has been tested

File: ehsm_kms_service/function.js [hunk 2]
```diff
@@ -264,8 +264,10 @@ const enroll_user_info = (action, DB, res, req) => {
   if (napi_res) {
     const { appid, apikey } = napi_res.result
     let cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
+    let sm_default_cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
     if (cmk_res) {
```
Comment (yang8621) [inline ehsm_kms_service/function.js:268]: Does the below change make sense?
if (sm_default_cmk_res && cmk_res)

Comment (Yidingluo1) [reply ehsm_kms_service/function.js:268]: > Does the below change make sense? if (sm_default_cmk_res && cmk_res)

I think if you want to make cmk different from defaultcmk, you must write it separately

Comment (wanghouqi) [reply ehsm_kms_service/function.js:268]: the if (cmk_res) will be changed to if (sm_default_cmk_res && cmk_res)

**Review by yang8621 — APPROVED** [review]

File: ehsm_kms_service/function.js (hunk 1 above)
Comment (syan10) [reply ehsm_kms_service/function.js:267]: generally, the abbreviation for "result" is "ret" not "res"!

anyway, you can change it later.

File: ehsm_kms_service/function.js [hunk 3]
```diff
@@ -264,8 +264,10 @@ const enroll_user_info = (action, DB, res, req) => {
   if (napi_res) {
     const { appid, apikey } = napi_res.result
     let cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
-    if (cmk_res) {
+    let sm_default_cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
+    if (cmk_res && sm_default_cmk_res) {
       const { cmk } = cmk_res.result
+      const sm_default_cmk = sm_default_cmk_res.result.cmk
```
Comment (syan10) [inline ehsm_kms_service/function.js:270]: add some comments here

Comment (wanghouqi) [reply ehsm_kms_service/function.js:270]: will be add: // create a default secret manager CMK for current appid.

**Review by syan10 — APPROVED** [review]

**Pull request merged**